### PR TITLE
provisioning grafana config

### PR DESCRIPTION
**Pull request by 2Fraennk**
- add initial grafana...
-- dashboards (opensearch, postgresql, mongodb)
-- datasources (prometheus, loki, jaeger)

### CODE REVIEW

File: config/grafana-provisioning/datasources/default.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    type: testdata
+  - name: Prometheus
+    type: prometheus
+    access: server
+    url: http://fem_prometheus:9090
+    editable: true
+  - name: Jaeger
+    type: jaeger
+    access: direct
+    url: http://fem_jaeger:16686
```
Comment (ronnyfriedland) [inline config/grafana-provisioning/datasources/default.yml:14]: Should be `fem_jaeger-query`

**Review by ronnyfriedland — CHANGES_REQUESTED** [review]
Please check jaeger datasource

File: config/grafana-provisioning/datasources/default.yml (hunk 1 above)
Comment (2Fraennk) [reply config/grafana-provisioning/datasources/default.yml]: parameter fixed

**Review by ronnyfriedland — APPROVED** [review]
LGTM

**Pull request merged**